core/Cargo.toml

```
@@ -59,9 +59,10 @@ serde_json = "1.0"
 prost-build = "0.11"
 
 [features]
-default = [ "secp256k1", "ecdsa" ]
+default = [ "secp256k1", "ecdsa", "rsa" ]
 secp256k1 = [ "libsecp256k1" ]
 ecdsa = [ "p256" ]
+rsa = [ "ring" ]
```

```suggestion
rsa = [ "dep:ring" ]
```

This should hide the `ring` feature that we are exposing through the new optional dependency.

---

Done!